src/views/Clients/ViewClients/index.jsx

```diff
@@ -155,6 +155,7 @@ export default class ViewClients extends PureComponent {
       >
         <Fragment>
           {loading && <Spinner loading />}
+          {error && error.graphQLErrors && <ErrorPanel error={error} />}
```

Comment:
Sorry, I think this line can be fully replaced with:

```jsx
<ErrorPanel error={error} />
```

Reply:
Oh okay that may be a previous pull request from someone else but I’ll change that!

Reply:
Ohh that was because of something I merged earlier on accident let me change that